src/shared/modules/auth/common.js

```diff
@@ -218,20 +217,16 @@ export const restoreSearchAndHashParams = () => {
 
     window.sessionStorage.setItem(AUTH_STORAGE_URL_SEARCH_PARAMS, '')
 
-    if (isObject(storedParams)) {
```

Comment:
Are we still handling the case where storedParams is NOT an object?

Reply:
Probably good to re-add this just in case